modules/preview-react/pill/lib/PillCount.tsx

```diff
@@ -1,39 +1,44 @@
 import React from 'react';
 
-import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
-import {Flex, FlexProps} from '@workday/canvas-kit-react/layout';
-import {borderRadius, colors, space} from '@workday/canvas-kit-react/tokens';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {FlexProps, mergeStyles} from '@workday/canvas-kit-react/layout';
+import {calc, createStencil, cssVar, px2rem} from '@workday/canvas-kit-styling';
+import {system} from '@workday/canvas-tokens-web';
 
 export interface PillCountProps extends FlexProps {}
 
-const StyledCountContainer = styled(Flex.as('span'))<StyledType>({
-  borderTopLeftRadius: borderRadius.zero,
-  borderTopRightRadius: borderRadius.m,
-  borderBottomLeftRadius: borderRadius.zero,
-  borderBottomRightRadius: borderRadius.m,
+export const pillCountStencil = createStencil({
+  base: {
+    borderTopLeftRadius: system.shape.zero,
+    borderTopRightRadius: system.shape.x1,
+    borderBottomLeftRadius: system.shape.zero,
+    borderBottomRightRadius: system.shape.x1,
+    borderTop: `${px2rem(1)} solid  ${cssVar(system.color.border.transparent)}`,
+    borderBottom: `${px2rem(1)} solid ${cssVar(system.color.border.transparent)}`,
+    borderRight: `${px2rem(1)} solid ${cssVar(system.color.border.transparent)}`,
+    display: 'inline-flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    height: px2rem(22),
+    width: px2rem(22),
+    padding: `0 ${cssVar(system.space.x1)}`,
+    marginInlineEnd: calc.multiply(system.space.x2, '-1'),
```

Comment:
Instead of using multiply you can use negate. This goes for any time you need to multiply a number by -1.

This way you don't have to use a multiply with a -1.

```suggestion
    marginInlineEnd: calc.negate(system.space.x2),
```